addons/account/wizard/account_move_reversal.py

```diff
@@ -121,6 +122,9 @@ def reverse_moves(self, is_modify=False):
         moves_to_redirect = self.env['account.move']
         for moves, default_values_list, is_cancel_needed in batches:
             new_moves = moves._reverse_moves(default_values_list, cancel=is_cancel_needed)
+            moves._message_log_batch(
+                bodies=dict((move.id, escape(_('This entry has been %s')) % reverse._get_html_link(title="reversed")) for move, reverse in zip(moves, new_moves))
```

Comment:
I don't think this is easy for translators. (and is `reversed` translated?)
Wouldn't it be safer to just put the link on the move name instead?

Reply:
If i remember correctly we did that because sometimes the moves has not yet a name and so the message "This entry has been reversed NAME_MOVE" would be something like "This entry has been reversed /" and the link would be put on the "/"